bin/meta

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 const chalk = require('chalk');
-const debug = require('debug')('meta');
```

Comment:
Can you nuke this change, please?

Reply:
Ping ping on this change `const debug = require('debug')('@essential-projects/meta');`. Can you reset it to `meta`?

Reply:
Sorry totally missed this